fdbserver/LogSystem.h

```diff
@@ -866,7 +872,9 @@ struct ILogSystem {
 	    int8_t primaryLocality,
 	    int8_t remoteLocality,
 	    std::vector<Tag> const& allTags,
-	    Reference<AsyncVar<bool>> const& recruitmentStalled) = 0;
+	    Reference<AsyncVar<bool>> const& recruitmentStalled,
+	    std::unordered_map<UID, std::vector<UID>> tLogGroupIdToServerIds,
```

Comment:
we may not need to pass these in since they can be set in LogSystemConfig which the returned ILogSytem contains.

Reply:
from my understanding it is created here https://github.com/apple/foundationdb/blob/partitioned-txns/fdbserver/TagPartitionedLogSystem.actor.cpp#L1658, so we need it in ILogSystem